src/ray/core_worker/common.h

```diff
@@ -285,11 +291,11 @@ template <>
 struct hash<ray::rpc::LineageReconstructionTask> {
   size_t operator()(const ray::rpc::LineageReconstructionTask &task) const {
     size_t hash = std::hash<std::string>()(task.name());
-    for (const auto &resource : task.resources()) {
-      hash ^= std::hash<std::string>()(resource.first);
-      hash ^= std::hash<double>()(resource.second);
-    }
     hash ^= std::hash<ray::rpc::TaskStatus>()(task.status());
+    for (const auto &label : task.labels()) {
```

Comment:
nit, should sort the label keys to produce consistent hash result, or just use an ordered_map.

Reply:
Actually we are using `^` so the order doesn't matter.